client/package.json

```diff
@@ -91,7 +93,8 @@
     "karma-spec-reporter": "^0.0.32",
     "karma-webpack": "^2.0.4",
     "node-sass": "^4.5.3",
-    "optimize-css-assets-webpack-plugin": "^3.1.1",
+    "optimize-css-assets-webpack-plugin": "^5.0.0",
```

Comment:
Doesn't technically need to be updated, I was trying to re-enable it but it still wasn't working. Will try again in a follow up PR.